src/app/public/modules/progress-indicator/progress-indicator.component.spec.ts

```diff
@@ -20,387 +20,649 @@ import {
 
 import {
   SkyProgressIndicatorDisplayMode,
-  SkyProgressIndicatorMessageType
+  SkyProgressIndicatorItemStatus,
+  SkyProgressIndicatorMessageType,
+  SkyProgressIndicatorNavButtonType
 } from './types';
 
-describe('Progress indicator component', function () {
+import {
+  SkyProgressIndicatorComponent
+} from './progress-indicator.component';
```

Comment:
I double-checked the imports and they appear in the correct order. Any more specifics?

Reply:
Shouldn't './progress-indicator.component' come before `./types`?

Reply:
Alphabetical ordering ☝️